doc/source/BuildingRunningTesting/BuildRunLandDA.rst

```diff
@@ -0,0 +1,238 @@
+
+.. code-block:: console
+
+   uw rocoto realize --input-file land_analysis.yaml --output-file land_analysis.xml
+
+If the command runs without problems, ``uwtools`` will output a "0 errors found" message similar to the following: 
+
+.. code-block:: console
+
+   [2024-03-01T20:36:03]     INFO 0 UW schema-validation errors found
+   [2024-03-01T20:36:03]     INFO 0 Rocoto validation errors found
+
+Run the Experiment
+********************
+
+To run the experiment, issue a ``rocotorun`` command from the ``parm`` directory: 
+
+.. code-block:: console
+
+   rocotorun -w land_analysis.xml -d land_analysis.db
+
+.. _VerifySuccess:
+
+Track Experiment Status
+========================
+
+To view the experiment status, run: 
+
+.. code-block:: console
+
+   rocotostat -w land_analysis.xml -d land_analysis.db
+
+If ``rocotorun`` was successful, the ``rocotostat`` command will print a status report to the console. For example:
+
+.. code-block:: console
+
+   CYCLE              TASK                 JOBID        STATE   EXIT STATUS   TRIES   DURATION
+   ======================================================================================================
+   200001030000    prepexp   druby://hfe08:41879   SUBMITTING             -       2        0.0
+   200001030000    prepobs                     -            -             -       -          -
+   200001030000   prepbmat                     -            -             -       -          -
+   200001030000     runana                     -            -             -       -          -
+   200001030000    runfcst                     -            -             -       -          -
+
+Users will need to issue the ``rocotorun`` command multiple times. The tasks must run in order, and ``rocotorun`` initiates the next task once its dependencies have completed successfully. Note that the status table printed by ``rocotostat`` only updates after each ``rocotorun`` command. For each task, a ``slurm-########.out`` log file is generated. A task that runs successfully will include a message with ``exit code 0:0`` at the bottom of the file: 
+
+.. code-block:: console
+
+   _______________________________________________________________
+   Start Epilog on node h24c45 for job 56463665 :: Fri Mar  1 22:38:34 UTC 2024
+   Job 56463665 finished for user Gillian.Petro in partition hera with exit code 0:0
+   _______________________________________________________________
+   End Epilogue Fri Mar  1 22:38:34 UTC 2024
+
+The experiment has successfully completed when all tasks say SUCCEEDED under STATE. Other potential statuses are: QUEUED, SUBMITTING, RUNNING, and DEAD. Users may view the ``slurm-########.out`` files to determine why a task may have failed. 
+
+Check Experiment Output
+=========================
+
+As the experiment progresses, it will create an experiment directory in ``$LANDDAROOT/landda_expts/EXP_NAME`` to hold experiment output. (Note that ``$EXP_NAME`` was set in ``land_analysis.yaml``.)
```

Comment:
Thanks! Is there any way to get those `slurm-########.out` files into a log directory (e.g., in `$LANDDAROOT/landda_expts/EXP_NAME/log` or something similar)? Not sure how difficult a change that would be, but I think it would be useful to have them there eventually instead of polluting the `parm` directory. Would also be good to have them named with the task they were running (e.g., `run_ana.out` instead of `slurm-########.out`).

Reply:
Good point again !! :) I think there should be. I'll update it too (if possible :)).